### PR TITLE
Add ColorSchemeUnit support

### LINKED ISSUE #59: Enhance color scheme testing
I'd appreciate the `develop` branch of @gerardroche's fork being merged into `master` here.

Compare: <https://github.com/randy3k/UnitTesting/compare/master...gerardroche:develop>

---

> This is for me to keep track of events...

### PR DESCRIPTION
**Pull request by gerardroche**
ColorSchemeUnit is a testing framework for Sublime Text color schemes:

https://github.com/gerardroche/sublime-color-scheme-unit

Example usage:

https://github.com/gerardroche/sublime-monokai-free/blob/master/.travis.yml

There are several issues of note about the implementation:

* The bootstrap must be call with the option `--with-color-scheme-unit`
option. The reason for this is to avoid unnecessary download of the
plugin for test suites that don't test any color schemes.

* Several log message format issues have been fixed.

* An edge-case issue where all arguments are not passed to the
`run_tests.py` script has been fixed.

Closes #59

### CODE REVIEW

File: sbin/travis.sh [hunk 1]
```diff
@@ -45,6 +51,8 @@ Bootstrap() {
 
         echo "download UnitTesting tag: $UNITTESTING_TAG"
         git clone --quiet --depth 1 --branch $UNITTESTING_TAG "$UT_URL" "$UT_PATH"
+        git -C "$UT_PATH" rev-parse HEAD
```
Comment (randy3k) [inline sbin/travis.sh]: why we need this?

File: sbin/travis.sh [hunk 2]
```diff
@@ -61,9 +69,33 @@ Bootstrap() {
 
         echo "download sublime-coverage tag: $COVERAGE_TAG"
         git clone --quiet --depth 1 --branch $COVERAGE_TAG "$COV_URL" "$COV_PATH"
+        git -C "$COV_PATH" rev-parse HEAD
```
Comment (randy3k) [inline sbin/travis.sh]: same

File: .travis.yml [hunk 3]
```diff
@@ -28,7 +28,7 @@ before_install:
       fi
 
 install:
-    - sh sbin/travis.sh bootstrap
+    - sh sbin/travis.sh bootstrap --with-color-scheme-unit
```
Comment (randy3k) [inline .travis.yml]: Maybe we just install `color-scheme-unit` anyway, it seems too much configurations to me.

Comment (randy3k) [inline .travis.yml]: may be we should prepare a command like `sh sbin/travis.sh install_color_scheme_unit` instead?

**Comment by randy3k** [comment]
merged to master.

I have removed those `rev-parse HEAD` lines and

and modify a bit the bootstrap process

https://github.com/randy3k/UnitTesting/blob/master/.travis.yml#L32

File: sbin/travis.sh (hunk 1 above)
Comment (gerardroche) [reply sbin/travis.sh]: When you checkout a branch rather than a tag it doesn't display the commit hash that is checked out. Knowing the commit hash is very useful for debugging.

File: .travis.yml (hunk 3 above)
Comment (gerardroche) [reply .travis.yml]: If you prefer, sure. It would probably make more sense as `sh sbin/travis.sh install_color_scheme_unit`

**Comment by gerardroche** [comment]
Having thought about it I don't think it's a good idea to install ColorSchemeUnit *after* ST is installed. The bootstrap installs UnitTesting and coverage packages, then ST is installed and killed (although debugging #54 I'm not so sure ST is being killed, at least on Linux). To avoid any potential issues I think it might be better is ColorSchemeUnit is installed before ST.

**Comment by randy3k** [comment]
I think it is in general safe to install packages after bootstrapping. I gave up the previous approach: instead of polling api status, we actively inject the scheduler code to sublime #61

**Comment by gerardroche** [comment]
This works for me now on Linux and Travis CI.
